scipy/stats/_morestats.py

```diff
@@ -855,20 +855,20 @@ def ppcc_plot(x, a, b, dist='tukeylambda', plot=None, N=80):
     return svals, ppcc
 
 
-def _log_mean(logx):
+def _log_mean(logx, axis):
     # compute log of mean of x from log(x)
-    return special.logsumexp(logx, axis=0) - np.log(len(logx))
+    return special.logsumexp(logx, axis=axis) - np.log(len(logx))
```

Comment:
```suggestion
    return special.logsumexp(logx, axis=axis) - np.log(logx.shape[axis])
```
Might look like it would only work for integer `axis` (not `None` or a tuple of ints), but the decorator standardizes the problem so that `axis` is always an integer (e.g. for `None`, ravel the inputs and `axis=0`).